server/public/layout.css

```diff
@@ -145,6 +156,17 @@
     background-color: white;
     border-radius: 10px;
   }
+
+  .saved-prompt {
+    color: white;
+    background-color: rgb(0 0 0 / 75%);
+    width: 360px;
+    padding: 8px 0;
+    font-size: 1.2rem;
+    position: absolute;
+    border-radius: 0 0 10px 10px;
+    transition: 0.2s;
```

Comment:
You only need to write the declarations that you wish to override. Everything that stays the same between your default styles and media query styles can be omitted. Double-check the rest of your code to see if there are any CSS declarations you can delete from your media queries.